log4j-slf4j-impl/src/main/java/org/apache/logging/slf4j/Log4jMDCAdapter.java

```diff
@@ -52,8 +52,7 @@ public void clear() {
     }
 
     @Override
-    @SuppressWarnings("unchecked") // nothing we can do about this, restricted by SLF4J API
-    public void setContextMap(@SuppressWarnings("rawtypes") final Map map) {
+    public void setContextMap(final Map<String, String> map) {
```

Comment:
This will break source compatibility, does it matter?

Reply:
thanks for your time! :-) 

Given that SLF4J interface is: 
```
    public void setContextMap(Map<String, String> contextMap);
```

Reply:
Seems ok then. Thank you for the info.